### PR TITLE
chore: Upgrade - Localizable Sample to run with latest uno and latest…

### PR DESCRIPTION
**Pull request by aesteves900**
… VS 2022

### CODE REVIEW

**Comment by aesteves900** [comment]
@jeromelaban, I consistently get the error below because there is no compatible version of Uno.UniversalImageLoader is compatible with UWP. What is the correct way to figure out this?

The reference comes from the Androd project.

NU1202: Package Uno.UniversalImageLoader 1.9.35 is not compatible with uap10.0.17134 (UAP,Version=v10.0.17134) / win10-x64. Package Uno.UniversalImageLoader 1.9.35 supports:
  - monoandroid10.0 (MonoAndroid,Version=v10.0)
  - net6.0-android30.0 (.NETCoreApp,Version=v6.0)
NU1202: Package Uno.UniversalImageLoader 1.9.35 is not compatible with uap10.0.17134 (UAP,Version=v10.0.17134) / win10-arm. Package Uno.UniversalImageLoader 1.9.35 supports:
  - monoandroid10.0 (MonoAndroid,Version=v10.0)
  - net6.0-android30.0 (.NETCoreApp,Version=v6.0)
NU1202: Package Uno.UniversalImageLoader 1.9.35 is not compatible with uap10.0.17134 (UAP,Version=v10.0.17134). Package Uno.UniversalImageLoader 1.9.35 supports:
  - monoandroid10.0 (MonoAndroid,Version=v10.0)
  - net6.0-android30.0 (.NETCoreApp,Version=v6.0)
NU1202: Package Uno.UniversalImageLoader 1.9.35 is not compatible with uap10.0.17134 (UAP,Version=v10.0.17134) / win10-arm64-aot. Package Uno.UniversalImageLoader 1.9.35 supports:
  - monoandroid10.0 (MonoAndroid,Version=v10.0)
  - net6.0-android30.0 (.NETCoreApp,Version=v6.0)
NU1202: Package Uno.UniversalImageLoader 1.9.35 is not compatible with uap10.0.17134 (UAP,Version=v10.0.17134) / win10-arm-aot. Package Uno.UniversalImageLoader 1.9.35 supports:
  - monoandroid10.0 (MonoAndroid,Version=v10.0)
  - net6.0-android30.0 (.NETCoreApp,Version=v6.0)
NU1202: Package Uno.UniversalImageLoader 1.9.35 is not compatible with uap10.0.17134 (UAP,Version=v10.0.17134) / win10-x64-aot. Package Uno.UniversalImageLoader 1.9.35 supports:
  - monoandroid10.0 (MonoAndroid,Version=v10.0)
  - net6.0-android30.0 (.NETCoreApp,Version=v6.0)
NU1202: Package Uno.UniversalImageLoader 1.9.35 is not compatible with uap10.0.17134 (UAP,Version=v10.0.17134) / win10-x86. Package Uno.UniversalImageLoader 1.9.35 supports:
  - monoandroid10.0 (MonoAndroid,Version=v10.0)

**Comment by jeromelaban** [comment]
> @jeromelaban, I consistently get the error below because there is no compatible version of Uno.UniversalImageLoader is compatible with UWP. What is the correct way to figure out this?
> 
> The reference comes from the Androd project.
> 
> NU1202: Package Uno.UniversalImageLoader 1.9.35 is not compatible with uap10.0.17134 (UAP,Version=v10.0.17134) / win10-x64. Package Uno.UniversalImageLoader 1.9.35 supports:
> 
> * monoandroid10.0 (MonoAndroid,Version=v10.0)
> * net6.0-android30.0 (.NETCoreApp,Version=v6.0)
>   NU1202: Package Uno.UniversalImageLoader 1.9.35 is not compatible with uap10.0.17134 (UAP,Version=v10.0.17134) / win10-arm. Package Uno.UniversalImageLoader 1.9.35 supports:
> * monoandroid10.0 (MonoAndroid,Version=v10.0)
> * net6.0-android30.0 (.NETCoreApp,Version=v6.0)
>   NU1202: Package Uno.UniversalImageLoader 1.9.35 is not compatible with uap10.0.17134 (UAP,Version=v10.0.17134). Package Uno.UniversalImageLoader 1.9.35 supports:
> * monoandroid10.0 (MonoAndroid,Version=v10.0)
> * net6.0-android30.0 (.NETCoreApp,Version=v6.0)
>   NU1202: Package Uno.UniversalImageLoader 1.9.35 is not compatible with uap10.0.17134 (UAP,Version=v10.0.17134) / win10-arm64-aot. Package Uno.UniversalImageLoader 1.9.35 supports:
> * monoandroid10.0 (MonoAndroid,Version=v10.0)
> * net6.0-android30.0 (.NETCoreApp,Version=v6.0)
>   NU1202: Package Uno.UniversalImageLoader 1.9.35 is not compatible with uap10.0.17134 (UAP,Version=v10.0.17134) / win10-arm-aot. Package Uno.UniversalImageLoader 1.9.35 supports:
> * monoandroid10.0 (MonoAndroid,Version=v10.0)
> * net6.0-android30.0 (.NETCoreApp,Version=v6.0)
>   NU1202: Package Uno.UniversalImageLoader 1.9.35 is not compatible with uap10.0.17134 (UAP,Version=v10.0.17134) / win10-x64-aot. Package Uno.UniversalImageLoader 1.9.35 supports:
> * monoandroid10.0 (MonoAndroid,Version=v10.0)
> * net6.0-android30.0 (.NETCoreApp,Version=v6.0)
>   NU1202: Package Uno.UniversalImageLoader 1.9.35 is not compatible with uap10.0.17134 (UAP,Version=v10.0.17134) / win10-x86. Package Uno.UniversalImageLoader 1.9.35 supports:
> * monoandroid10.0 (MonoAndroid,Version=v10.0)

The issue is in the original code:
https://github.com/unoplatform/Uno.Samples/blob/c4bf9428de1cdcaf349f40a1dd125d0d8768e251/UI/LocalizationSamples/Localization/UnoLocalization.UWP/UnoLocalization.Uwp.csproj#L16

You can remove the line as this package is not compatible with UWP.

**Comment by aesteves900** [comment]
@jeromelaban , I applied the same solution from another project's updating System.Memory NuGet packaged, but with these two projects with no success. Do you have any clue about that? 


UnoLocalization.Droid
UnoLocalization.Tizen

error : Generation failed: System.MissingMethodException: Method not found: 'System.ReadOnlySpan`1<Char> Microsoft.IO.Path.GetFileName(System.ReadOnlySpan`1<Char>)'. [D:\a\1\s\UI\LocalizationSamples\Localization\UnoLocalization.Skia.Tizen\UnoLocalization.Skia.Tizen.csproj]
MSBUILD : error :    at Microsoft.Build.Shared.FileMatcher.IsFileNameMatch(String path, String pattern) [D:\a\1\s\UI\LocalizationSamples\Localization\

**Comment by jeromelaban** [comment]
This particular error only happens when Uno has not been updated to latest. Make sure there are no leftovers from previous versions.

**Comment by jeromelaban** [comment]
You're missing an update in the tizen project:

  MSBUILD : error : Generation failed: System.MissingMethodException: Method not found: 'System.ReadOnlySpan`1<Char> Microsoft.IO.Path.GetFileName(System.ReadOnlySpan`1<Char>)'. [D:\a\1\s\UI\LocalizationSamples\RuntimeCultureSwitching\UnoLocalization.Skia.Tizen\UnoLocalization.Skia.Tizen.csproj]

File: UI/LocalizationSamples/RuntimeCultureSwitching/UnoLocalization.Skia.Tizen/UnoLocalization.Skia.Tizen.csproj [hunk 1]
```diff
@@ -20,7 +20,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="5.0.0" />
-    <PackageReference Include="SkiaSharp.Views" Version="2.80.3" />
+    <PackageReference Include="SkiaSharp.Views" Version="2.88.3" />
     <PackageReference Include="Uno.UI.Skia.Tizen" Version="4.1.9" />
```
Comment (jeromelaban) [inline UI/LocalizationSamples/RuntimeCultureSwitching/UnoLocalization.Skia.Tizen/UnoLocalization.Skia.Tizen.csproj:24]: You'll need to update these packages as well.

File: UI/LocalizationSamples/Localization/UnoLocalization.Shared/App.xaml [hunk 2]
```diff
@@ -6,8 +6,7 @@
     
     <Application.Resources>
         <ResourceDictionary>
-            <ResourceDictionary.MergedDictionaries>
-                <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
```
[the remark this answers is not in the document]
Comment (aesteves900) [reply UI/LocalizationSamples/Localization/UnoLocalization.Shared/App.xaml:10]: I made a mistake. The code is back up. Thanks!

**Review by jeromelaban — APPROVED** [review]
Great!

**Pull request merged**